site/_includes/_footer.liquid

```diff
@@ -39,10 +39,11 @@
       <div class="col-xs-12 col-md-6 col-lg-3">
         <h2>Connect with Us</h2>
         <ul class="inline">
-          <li><a href="https://github.com/fusionauth" target="_blank"><i class="fab fa-github-square fa-3x"></i></a></li>
-          <li><a href="https://twitter.com/fusionauth" target="_blank"><i class="fab fa-twitter-square fa-3x"></i></a></li>
-          <li><a href="https://facebook.com/fusionauth" target="_blank"><i class="fab fa-facebook-square fa-3x"></i></a></li>
-          <li><a href="https://www.linkedin.com/company/fusionauth/" target="_blank"><i class="fab fa-linkedin fa-3x"></i></a></li>
+          <li><a href="https://github.com/fusionauth" target="_blank"><i class="fab fa-github-square fa-2x"></i></a></li>
+          <li><a href="https://twitter.com/fusionauth" target="_blank"><i class="fab fa-twitter-square fa-2x"></i></a></li>
+          <li><a href="https://facebook.com/fusionauth" target="_blank"><i class="fab fa-facebook-square fa-2x"></i></a></li>
+          <li><a href="https://www.linkedin.com/company/fusionauth/" target="_blank"><i class="fab fa-linkedin fa-2x"></i></a></li>
+          <li><a href="https://fusionauth.io/blog/feed.xml" target="_blank"><i class="fas fa-rss-square fa-2x"></i></a></li>
```

Comment:
Did you mean to change the icon size? `fa-3x` --> `fa-2x`?

Reply:
I did. Otherwise the 5th icon wraps the row onto two lines.

Reply:
Hmm. Ok. This changes the style for those icons, perhaps we just need to make some additional space in that row. 

@voidmain is doing some site / account work at the moment - any opinion if we make these smaller or widen the spot for these social icons.

Reply:
I'm going to manage the layout in style and remove the `fa-2x`. I'll merge this down and then fix it locally on master.